frontend/javascripts/test/_force-exit.mjs

```diff
@@ -0,0 +1,8 @@
+// See https://github.com/avajs/ava/blob/main/docs/08-common-pitfalls.md#timeouts-because-a-file-failed-to-exit
+
+import process from "node:process";
+import { registerCompletionHandler } from "ava";
+
+registerCompletionHandler(() => {
+  process.exit();
```

Comment:
Ava used to do this automatically in earlier versions. I tried to properly clean up after the tests by canceling the sagas which worked and avoided ava timeout errors but it took ~15s for the process to exit, anyways. I decided not to investigate further and instead follow the docs.